### PR TITLE
Update docs for release 0.8.4

### PR DESCRIPTION
**Pull request by karol-brejna-i**
* Updated version number
* Fixed description of updating 'latest' folder
* Regenerated the contents

### CODE REVIEW

File: content/community.html [hunk 1]
```diff
@@ -112,22 +112,22 @@ <h2 id="mailing-lists">Mailing Lists</h2>
   <tbody>
     <tr>
       <td>User support and questions mailing list<br /><strong>user</strong>@gearpump.incubator.apache.org</td>
-      <td><a href="mailto:user-subscribe@gearpump.incubator.apache.org?subject=Subscribe">Subscribe</a></td>
-      <td><a href="mailto:user-unsubscribe@gearpump.incubator.apache.org?subject=Unsubscribe">Unsubscribe</a></td>
-      <td><a href="mailto:user@gearpump.incubator.apache.org">Compose</a></td>
+      <td><a href="&#109;&#097;&#105;&#108;&#116;&#111;:&#117;&#115;&#101;&#114;&#045;&#115;&#117;&#098;&#115;&#099;&#114;&#105;&#098;&#101;&#064;&#103;&#101;&#097;&#114;&#112;&#117;&#109;&#112;&#046;&#105;&#110;&#099;&#117;&#098;&#097;&#116;&#111;&#114;&#046;&#097;&#112;&#097;&#099;&#104;&#101;&#046;&#111;&#114;&#103;&#063;&#115;&#117;&#098;&#106;&#101;&#099;&#116;&#061;&#083;&#117;&#098;&#115;&#099;&#114;&#105;&#098;&#101;">Subscribe</a></td>
```
Comment (manuzhang) [inline content/community.html]: what are these codes ?

**Comment by manuzhang** [comment]
R: @manuzhang

File: content/community.html (hunk 1 above)
Comment (karol-brejna-i) [reply content/community.html]: These are strange.  THese are ascii codes for href value. 
But this works: the link still produces proper "mailto: xxx" links.
I am not sure what caused the change... And I am not sure how to force the tools to leave it be...

But like I said, it works anyhow.

Comment (karol-brejna-i) [reply content/community.html]: while testing: these produce proper href entries...

File: releases/README.md [hunk 2]
```diff
@@ -9,5 +9,5 @@ directory structure
   |-- ...
 ```
 
-For each release, please run command "docs/build_doc.sh" under Gearpump repository
-and copy the whole `docs/_site` directory under release and rename `_site` directory to corresponding version number. 
+For each release, please run command "docs/build_doc.sh" under Gearpump repository, rename `latest` directory to name reflecting previous version number
```
Comment (manuzhang) [inline releases/README.md]: Maybe we should automate this in the future by asking user for `GEARPUMP_HOME`

File: releases/README.md [hunk 3]
```diff
@@ -9,5 +9,6 @@ directory structure
   |-- ...
 ```
 
-For each release, please run command "docs/build_doc.sh" under Gearpump repository, rename `latest` directory to name reflecting previous version number
-and copy the whole `docs/site` directory (from gearpump repo, after building) to release and rename `site` directory `latest`. 
+For each release, please run the command "docs/build_doc.sh" under Gearpump repository, 
+rename `latest` directory to a name reflecting previous version number 
+and copy the whole `docs/site` directory (from Gearpump repo, after having it built) to release and rename `site` directory to `latest`.
```
Comment (manuzhang) [inline releases/README.md]: Maybe we should automate this in the future by asking user for GEARPUMP_HOME

File: content/community.html (hunk 1 above)
Comment (manuzhang) [reply content/community.html]: this shouldn't be changed since it is not per release

Comment (manuzhang) [reply content/community.html]: are they generated by jekyll ?

Comment (karol-brejna-i) [reply content/community.html]: It looks like it. What's strange, it didn't happen before.

File: releases/README.md (hunk 3 above)
Comment (karol-brejna-i) [reply releases/README.md]: Yes, we should. Actually, there is a lot of confusion with the dependencies needed for building the docs and the commands themselves.

I am starting to experiment with dockers for building gearpump and the docs: https://github.com/karol-brejna-i/gearpump-building-docker

These would encapsulate the dependencies, the commands, etc.

File: content/community.html (hunk 1 above)
Comment (manuzhang) [reply content/community.html]: It works normally for me. Have you generated this every time ?

**Comment by manuzhang** [comment]
@karol-brejna-i Could you also help to remove https://github.com/apache/incubator-gearpump-site/tree/asf-site/content/2015/12/29 ? There is no reason for it to exist but html-proofer finds a lot of broken links there.

**Comment by manuzhang** [comment]
That's actually generated by https://github.com/apache/incubator-gearpump-site/blob/asf-site/_posts/2015-12-29-0.7.1_release.md

**Comment by manuzhang** [comment]
LGTM

**Comment by karol-brejna-i** [comment]
Let me try to deal with 0.7.1_release.md, too.

https://jekyllrb.com/docs/posts/ - it looks like this is jekyll's mechanism for creating blog-like posts. Removing the file you mentioned should fix the issue.

**Comment by manuzhang** [comment]
Note this is already merged at https://git-wip-us.apache.org/repos/asf?p=incubator-gearpump-site.git;a=commit;h=f2a69d38efa9a4415d78fca68868f176ad60dd49. The big change seems to choke the sync every time.

**Pull request closed without merging**